README.md

```diff
@@ -1,10 +1,57 @@
-SET UP ENVIRONMENT VARIABLES
+# Northcoders News API
 
-When cloning this repo and running the project locally please create two seperate .env files.
+## Background
 
-.env.test and .env.development
+We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
 
-They should then be populated with the correct database information.
+Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
 
-.env.test should be populated with PGDATABASE=nc_news_test.
-.env.development should be populated with PGDATABASE=nc_news
+## Kanban
```

Comment:
the README should contain instructions on how to run your code.

We don't need this information about the trello board/task in here